src/modules/core-milestone/core-milestone.tsx

```diff
@@ -0,0 +1,289 @@
+marginLeft: ${props =>
+  props.isWhite === true ? `${minorScale(1)}px` : `${minorScale(1)}px`}};;
+`;
+
+//left has attended class 301
+const StyledLeftHasAttendedClass301Wrapper = Styled.View<IProps>`
+height: ${Number(minorScale(1)) + 1}px;
+width: ${Number(minorScale(1)) + 1}px;
+margin-right: -1px;
+background-color:${props =>
+  props.hasAttendedClass301 && props.hasSignedMinistryCovenant
+    ? props.isWhite === true
+      ? '#fff'
+      : props.colorMilestone
+    : props.hasAttendedClass301
+    ? 'transparent'
+    : '#dde1e4'};
+borderRadius: 1px;
+borderWidth: 0.5px;
+border-color: ${props =>
+  props.hasAttendedClass301 && props.hasSignedMinistryCovenant
+    ? 'transparent'
+    : props.hasAttendedClass301
+    ? props.isWhite === true
+      ? '#fff'
+      : props.colorMilestone
+    : 'transparent'};
+transform: rotate(135deg);
+`;
+
+//Top hasAttendedClass201
+const StyledClasshasAttendedClass201Wrapper = Styled.View<IProps>`
+            height: ${Number(minorScale(1)) + 1}px;
+              width: ${Number(minorScale(1)) + 1}px;
+              backgroundColor:${props =>
+                props.hasAttendedClass201 && props.hasSignedMaturityCovenant
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : props.hasAttendedClass201
+                  ? 'transparent'
+                  : '#dde1e4'};
+              borderRadius: 1px;
+              borderWidth: 0.5px;
+              borderColor:${props =>
+                props.hasAttendedClass201 && props.hasSignedMaturityCovenant
+                  ? 'transparent'
+                  : props.hasAttendedClass201
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : 'transparent'};
+              transform:rotate(135deg);
+`;
+
+// Bottom hasAttendedClass401
+const StyledClassHasAttendedClass401Wrapper = Styled.View<IProps>`
+height: ${Number(minorScale(1)) + 1}px;
+              width: ${Number(minorScale(1)) + 1}px;
+              marginTop: 4;
+              backgroundColor:${props =>
+                props.hasAttendedClass401 && props.hasSignedMissionCovenant
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : props.hasAttendedClass401
+                  ? 'transparent'
+                  : '#dde1e4'};
+              borderRadius: 1px;
+              borderWidth: 0.5px;
+              borderColor:${props =>
+                props.hasAttendedClass401 && props.hasSignedMissionCovenant
+                  ? 'transparent'
+                  : props.hasAttendedClass401
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : 'transparent'};
+              transform:rotate(135deg);
```

Comment:
Can we please avoid these repetitive statements?

Reply:
@ajaybushel 
it is different of  4 boxes for class core milestone